### PR TITLE
libbpg 0.9.7

### PR DESCRIPTION
**Pull request by bilhackmac**

### CODE REVIEW

File: Formula/libbpg.rb [hunk 1]
```diff
@@ -29,6 +29,8 @@ def install
     args << "USE_X265=" if build.without? "x265"
 
     system "make", "install", "prefix=#{prefix}", "CONFIG_APPLE=y", *args
+
+    share.install Dir["html/bpgdec*.js"]
```
Comment (MikeMcQuaid) [inline Formula/libbpg.rb]: How are these useful in `share`? I'm trying to figure out how they'd be used?

Comment (bilhackmac) [inline Formula/libbpg.rb]: I don't really know where to put these files. They must be copied to a web site to enable viewing bpg images in your browsers.

Comment (MikeMcQuaid) [inline Formula/libbpg.rb]: Cool. Just use `pkgshare` instead of `share`.

**Review by MikeMcQuaid — CHANGES_REQUESTED** [review]

**Comment by bilhackmac** [comment]
Update done

**Review by MikeMcQuaid — APPROVED** [review]

**Comment by nijikon** [comment]
You are awesome, thanks for your contribution! :tada:

**Pull request closed without merging**